packages/react-native-bridge/ios/Gutenberg.swift

```diff
@@ -186,6 +186,10 @@ public class Gutenberg: NSObject {
         sendEvent(.updateEditorSettings, body: settingsUpdates)
     }
 
+	public func showEditorHelp() {
+		bridgeModule.sendEventIfNeeded(.showEditorHelp, body: nil)
+	}
```

Comment:
It looks like the indentation may be off on these lines in comparison to other lines in this file. Not sure if GitHub is presenting it odd or if it is truly off.

Reply:
I'm not sure why github is displaying it this way, but I opened the file in VS Code and it's definitely not formatted that way:

![Screen Shot 2021-07-02 at 5 04 01 PM](https://user-images.githubusercontent.com/5810477/124328393-82524700-db57-11eb-9376-73abe840f4da.png)

Reply:
@dcalhoun you were correct about the spaces vs. tabs. I updated in 5189548855c49f940ceb2bc406dd0b2203c144c6 and that seems to have fixed it.